internal/storage/hub/bundle.go

```diff
@@ -239,29 +239,29 @@ func (b *Bundle) loadPolicySet(idHex, fileName string) (*runtimev1.RunnablePolic
 }
 
 func (b *Bundle) InspectPolicies(ctx context.Context, params storage.ListPolicyIDsParams) (map[string]*responsev1.InspectPoliciesResponse_Result, error) {
-	var fqns []string
+	var policyIDs []string
 	if len(params.IDs) == 0 {
 		var err error
-		if fqns, err = b.ListPolicyIDs(ctx, params); err != nil {
+		if policyIDs, err = b.ListPolicyIDs(ctx, params); err != nil {
```

Comment:
It should be `b.ListPolicyIDs` that should be changed to return the policy keys.  That'll make sure that the behaviour is consistent across the board for any existing or future code that uses `ListPolicyIDs` and does something with it.

Reply:
Oh sorry, I was confused by the diff.